test.sh

```diff
@@ -3,6 +3,10 @@
 echo "*** Running custom_active_record_observer specs"
 
 bundle install                                      || exit 1
+bundle exec appraisal install                       || exit 1
 bundle exec rake app:db:drop app:db:create          || exit 1
 bundle exec rake app:db:migrate app:db:test:prepare || exit 1
-bundle exec rspec spec
+
+bundle exec rspec spec --exclude-pattern 'spec/apprisals/**/*_spec.rb' &&
```

Comment:
doesn't `--exclude-pattern 'spec/apprisals/**'` work?

Reply:
it doesn't